templates/build/small-card-detail.build.hbs

```diff
@@ -54,12 +54,21 @@
         \{{#ifCond type '==' 'image'}}
           \{{#if content}}
             \{{#if labelEnabled}}
-            <div class="small-card-list-detail-label">\{{label}}</div>
+	          <div class="small-card-list-detail-label">\{{label}}</div>
```

Comment:
Incorrect indentation

Reply:
@YaroslavOvdii This is not addressed. I commented on it 25 days ago cc @sofiiakvasnevska